src/pykeen/nn/modules.py

```diff
@@ -964,6 +1040,7 @@ class SEInteraction(
     func = pkf.se_interaction
 
     @staticmethod
+    # docstr-coverage: inherited
```

Comment:
probably an artifact from the automated script, right? it's weird this is after the decorator

Reply:
27c8b86